login-workflow/src/screens/SiteOptionsScreen/SiteOptionsScreenBase.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { SiteOptionsScreenProps } from './types';
+import { WorkflowCard, WorkflowCardHeader, WorkflowCardBody, WorkflowCardInstructions } from '../../components';
+import Box from '@mui/material/Box';
+import CardActions from '@mui/material/CardActions';
+import Button from '@mui/material/Button';
+import Divider from '@mui/material/Divider';
+/**
+ * Component that renders a success screen
+ *
+ * @param icon the icon to be displayed on the screen
+ * @param message success message to be displayed on the screen
+ * @param joinExistingOrgLabel to display label for the custom button which redirects to existing organization flow
+ * @param createNewOrgLabel display label for the custom button which redirects to create a new organization flow
+ * @param onJoinExistingOrg function to call when user wants to join existing organization
+ * @param onCreateNewOrg function to call when user wants to create a new  organization
+ * @param WorkflowCardBaseProps props that will be passed to the WorkflowCard component
+ * @param WorkflowCardHeaderProps props that will be passed to the WorkflowCardHeader component
+ * @param WorkflowCardInstructionProps props that will be passed to the WorkflowCardInstructions component
+ *
+ * @category Component
+ */
+
+export const SiteOptionsScreenBase: React.FC<React.PropsWithChildren<SiteOptionsScreenProps>> = (props) => {
+    const {
+        icon,
+        message = '',
+        joinExistingOrgLabel = '',
+        createNewOrgLabel = '',
+        canCreateNewOrg,
+        canJoinExistingOrg,
+        onJoinExistingOrg,
+        onCreateNewOrg,
+    } = props;
+    const cardBaseProps = props.WorkflowCardBaseProps || {};
+    const headerProps = props.WorkflowCardHeaderProps || {};
+    const instructionsProps = props.WorkflowCardInstructionProps || {};
+    return (
+        <WorkflowCard {...cardBaseProps}>
+            <WorkflowCardHeader {...headerProps} />
+            {Object.keys(instructionsProps).length !== 0 && <WorkflowCardInstructions {...instructionsProps} />}
+            <WorkflowCardBody>
```

Comment:
should this be wrapped in <ErrorManager {...errorDisplayConfig}> and be imported

Reply:
Done thanks!!!